docs/user/integration/dynatrace/README.md

```diff
@@ -82,23 +82,22 @@ As a result, you see data arriving in your environment, advanced Kubernetes moni
 
 Next, you set up the ingestion of custom span and Istio span data, and, optionally, custom metrics based on OTLP.
 
-### Create Access Token
-
-To push custom metrics and spans to Dynatrace, set up an [API Token](https://docs.dynatrace.com/docs/manage/access-control/access-tokens).
-
-Follow the instructions in [Dynatrace: Generate an access token](https://docs.dynatrace.com/docs/manage/access-control/access-tokens#create-api-token) and select the following scopes:
+### Create Secret
+1. To push custom metrics and spans to Dynatrace, set up a [dataIngestToken](https://docs.dynatrace.com/docs/manage/access-control/access-tokens).
 
-- **Ingest metrics**
-- **Ingest OpenTelemetry traces**
+   Follow the instructions in [Dynatrace: Generate an access token](https://docs.dynatrace.com/docs/manage/access-control/access-tokens#create-api-token) and select the following scopes:
 
-### Create Secret
+   - **Ingest metrics**
+   - **Ingest OpenTelemetry traces**
 
-To create a new Secret containing your access token, replace the `{API_TOKEN}` placeholder with the token you created and run the following command:
+2. Create an [apitoken](https://docs.dynatrace.com/docs/manage/access-control/access-tokens) by selecting the template `Kubernetes: Dynatrace Operator`.
 
-```bash
-kubectl -n $DYNATRACE_NS create secret generic dynatrace-token --from-literal="apiToken=Api-Token {API_TOKEN}"
-```
+3. To create a new Secret containing your access tokens, replace the `{API_TOKEN}` and `{DATA_INGEST_TOKEN}` placeholder with the `apitoken` and `dataIngestToken` you created, replace the `{API_URL}` placeholder with the Dynatrace endpoint, and run the following command:
```

Comment:
The placeholders have different names in the Secret example and the kubectl example below